StudCity.Frontend/src/components/SearchUsersForCreatedChats/index.jsx

```diff
@@ -0,0 +1,172 @@
+import React, { useEffect, useState, useRef } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import Tabs from "../Tabs";
+import { LABELS_USERS_GROUPS } from "../../utils/constants";
+import Scroller from "../Scroller";
+import { selectDataUsersFound, getUsersSearch } from "../../app/features/userSlice";
+import TextField from "../../UI/fields/TextField";
+import Spinner from "../Spinner";
+import Avatar from "../../UI/Avatar";
+
+export default function UserGroupSearch({ setSelectedUsers, selectedUsers }) {
+	const [tabIndex, setTabIndex] = useState(0);
+	const dispatch = useDispatch();
+	const [users, setUsers] = useState([]);
+	const [searchData, setSearchData] = useState('');
+	const [totalItemsCount, setTotalItemsCount] = useState(0);
+	const { data, loading } = useSelector(selectDataUsersFound);
+	const [indexPage, setIndexPage] = useState(1);
+	const containerRef = useRef(null);
+
+	useEffect(() => {
+		if (containerRef.current) {
+			containerRef.current.addEventListener('scroll', handleScroll);
+		}
+		return () => {
+			if (containerRef.current) {
+				containerRef.current.removeEventListener('scroll', handleScroll);
+			}
+		};
+
+	}, [containerRef.current, loading])
+
+	useEffect(() => {
```

Comment:
I think for readability purpose you would better wright these if statements like this, also it would avoid unnecessary repetition of setting a state:
if (data?.items?.length === 0) setUsers([]);

if (data.items?.length > 0) {
   if (indexPage === 1) {
      setUsers(data.items);
      setTotalItemsCount(data.pageViewModel.totalItemsCount);
   }
   
   setUsers(prev => [...prev, ...data.items]);
}

Reply:
this problem I fixed in other PR